scripts/jenkins/ci.suse.de/openstack-mkcloud.xml

```diff
@@ -253,11 +253,18 @@ case $cloudsource in
     GM4|GM4+up|GM5|GM5+up)
         echo "Not setting want_ldap=1"
         ;;
+    develcloud6)
+        echo "Setting want_ldap=0 for upgrade jobs until hybrid backend is ported to Mitaka"
+        [[ $mkcloudtarget == "*plain_with_upgrade*" ]] &amp;&amp; export want_ldap=0
```

Comment:
or [[ $mkcloudtarget =~ upgrade ]] to also cover the case when someone gave the cloudupgrade step explicitly.